### ISSUE TITLE
Not compatible with Python 3

### ISSUE THREAD

**Issue by Zamiell** [phfaist/updater4pyi#4]
It should say in the README and the Docs that this is a legacy library that only runs on Python 2.

**Comment by phfaist** [comment]
Indeed, it's not compatible with Python 3, and it would be nice if it was. You're welcome to provide patches to make it run with python 3 as well.

**Comment by djhoese** [comment]
It looks like one of the forks has updated this to work with python 3, see https://github.com/chennan47/updater4pyi

Maybe @chennan47 could submit a pull request for this?

**Comment by phfaist** [comment]
That would be very welcome. (though after a brief glance, I couldn't tell if the code would still work under python 2? Ideally the code would run under both versions.)
